src/win32/posix_w32.c

```diff
@@ -130,7 +130,7 @@ int p_fsync(int fd)
 	return 0;
 }
 
-GIT_INLINE(time_t) filetime_to_time_t(const FILETIME *ft)
+__inline time_t filetime_to_time_t(const FILETIME *ft)
```

Comment:
However please add a nice prefix here when you do - something like `git__win32_filetime_to_time_t`